### ISSUE TITLE
Crash happens

### ISSUE THREAD

**Issue by Cha0sX** [Commit451/YouTubeExtractor#28]
```
java.lang.Exception: failed to find pattern "(encodeURIComponent=function\([a-zA-Z0-9_]+\)\{.+?\})
        at com.commit451.youtubeextractor.Util.matchGroup(Util.kt:22)
        at com.commit451.youtubeextractor.Util.matchGroup1(Util.kt:11)
        at com.commit451.youtubeextractor.JavaScriptUtil.loadDecryptionCode(JavaScriptUtil.kt:23)
        at com.commit451.youtubeextractor.YouTubeExtractor.parseVideoItags(YouTubeExtractor.kt:150)
        at com.commit451.youtubeextractor.YouTubeExtractor.parseVideoStreams(YouTubeExtractor.kt:127)
        at com.commit451.youtubeextractor.YouTubeExtractor.access$parseVideoStreams(YouTubeExtractor.kt:16)
        at com.commit451.youtubeextractor.YouTubeExtractor$extract$1.call(YouTubeExtractor.kt:79)
        at com.commit451.youtubeextractor.YouTubeExtractor$extract$1.call(YouTubeExtractor.kt:16)
        at io.reactivex.internal.operators.single.SingleDefer.subscribeActual(SingleDefer.java:36)
        at io.reactivex.Single.subscribe(Single.java:3438)
```

**Comment by Jawnnypoo** [comment]
Can you please give the video ID that you are trying to extract?

**Comment by naveedhassan913** [comment]
I made a lib for extracting yt videos.. And its working perfectly on any video.
https://github.com/naveedhassan913/YouTubeExtractor

**Comment by Jawnnypoo** [comment]
It is now fixed with `5.0.0`
